### PR TITLE
Calculate prior prob for variants after grey zone

### PR DESCRIPTION
**Pull request by ccdlott**
Also adds global variables for transcript data so API call is not necessary for every variant

### CODE REVIEW

File: pipeline/splicing/calcVarPriors.py [hunk 1]
```diff
@@ -245,7 +259,10 @@ def getExonBoundaries(variant):
     Uses function implemented in calcMaxEntScanMeanStd to get data for variant's transcript
     '''
     varTranscript = variant["Reference_Sequence"]
-    transcriptData = fetch_gene_coordinates(str(varTranscript))
+    if varTranscript == BRCA1_RefSeq:
```
Comment (zfisch) [inline pipeline/splicing/calcVarPriors.py]: the three identical blocks can all be a function call, e.g. something like:
```
def getTranscriptData(referenceSequence):
    if varTranscript == BRCA1_RefSeq: 
            return brca1TranscriptData
    elif varTranscript == BRCA2_RefSeq:
            return brca2TranscriptData
```

then just call it when you need it:
```
transcriptData = getTranscriptData(varTranscript)

**Comment by ccdlott** [comment]
@zfisch can I merge this now? Or do I need to wait until the PR is approved before merging?

File: pipeline/splicing/calcVarPriors.py [hunk 2]
```diff
@@ -245,8 +263,8 @@ def getExonBoundaries(variant):
     Uses function implemented in calcMaxEntScanMeanStd to get data for variant's transcript
     '''
     varTranscript = variant["Reference_Sequence"]
-    transcriptData = fetch_gene_coordinates(str(varTranscript))
-
+    transcriptData = getTranscriptData(varTranscript)
+
```
Comment (zfisch) [inline pipeline/splicing/calcVarPriors.py]: can rem white space

**Comment by zfisch** [comment]
@ccdlott wait for approval unless the change is crucial or blocking and without error -- best to be diligent as much as possible

**Pull request merged**